saskatoon/member/admin.py

```diff
@@ -279,6 +282,13 @@ def clean_column(col: Optional[str]) -> str:
                     f"Something went wrong: {e}"
                 )
 
+    @admin.action(description="Sets selected User(s)'s agreed_terms field to False")
+    def unset_agreed_terms(self, request, queryset):
+        for u in queryset:
+            u.agreed_terms = False
+            u.save()
```

Comment:
i which context would we need that? in case we update the terms?

Reply:
Btw it's better to avoid the for loop in such a case: `queryset.update(agreed_terms=False)`

Reply:
simone mentioned wanting to reset this field for everyone in case the agreement ever updates.

maybe i should rename to `reset_agreed_terms`

I just copied from other actions above, but def gonna go for queryset